pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -632,16 +632,15 @@ def process_circuits(
             else:
                 assert language == Language.QIR
                 warnings.warn(
-                    "Support for Language.QIR is experimental; this will probably fail!"
+                    "Support for Language.QIR is experimental; this may fail!"
                 )
-                if "pytket_to_qir" not in locals():
+                try:
```

Comment:
Do we want to add a check of the version of pytket-qir here as well?

Reply:
> Do we want to add a check of the version of pytket-qir here as well?

What should the check be?

Reply:
```suggestion
                try:
                    import pytket.qir.__extension_version__ as version
                    if (int(version.split(".")[1]) < 2):
                        raise Error("please update to pytket-qir >= 0.2.0")

Reply:
Something like this?

Reply:
Yes I had something in mind like this.